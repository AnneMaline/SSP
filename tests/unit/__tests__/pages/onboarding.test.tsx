import { render } from "@testing-library/react";
import OnboardingPage from "@/app/Onboarding/page";

describe("Onboarding Page", () => {
  it("renders correctly", () => {
    const { container } = render(<OnboardingPage />);
    expect(container).toMatchSnapshot();
  });
});
