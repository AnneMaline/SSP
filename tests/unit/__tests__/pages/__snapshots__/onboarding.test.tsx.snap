// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Onboarding Page renders correctly 1`] = `
<div>
  <div>
    <h1>
      Onboarding
    </h1>
  </div>
</div>
`;
